source/Server.Contracts/Endpoint/AgentlessEndpointResource.cs

```diff
@@ -0,0 +1,6 @@
+namespace Sashimi.Server.Contracts.Endpoint
+{
+    public abstract class AgentlessEndpointResource : EndpointResource
```

Comment:
I'm not sold on this name. There is still an agent, it's just a Worker. These are more like "DeploysViaWorkerDeploymentTarget"

Reply:
That's just how it was before, I think we can delete this intermediate class altogether

Reply:
I've delete it 🎉